tests/process.rs

```diff
@@ -24,115 +24,110 @@ fn test_process() {
+            .spawn()
+            .unwrap()
+    } else {
+        std::process::Command::new("sleep")
+            .arg("3")
+            .stdout(std::process::Stdio::null())
+            .spawn()
+            .unwrap()
+    };
+    std::thread::sleep(std::time::Duration::from_millis(250));
     let mut s = sysinfo::System::new();
     assert!(s.processes().is_empty());
     s.refresh_processes();
+    p.kill().ok();
     assert!(!s.processes().is_empty());
     if let Some(process) = s.process(p.id() as sysinfo::Pid) {
-        assert_eq!(process.cmd(), &["timeout", "/t", "3"]);
+        if cfg!(target_os = "windows") {
+            assert_eq!(process.cmd(), &["waitfor", "/t", "3", "CmdSignal"]);
+        } else {
+            assert_eq!(process.cmd(), &["sleep", "3"]);
+        }
     } else {
-        // We're very likely on a "linux-like" shell so let's try some unix command...
-        unix_like_cmd();
+        panic!("Process not found!");
     }
 }
 
 #[test]
-#[cfg(not(windows))]
-fn test_get_cmd_line() {
-    if sysinfo::System::IS_SUPPORTED && !cfg!(feature = "apple-sandbox") {
-        unix_like_cmd();
-    }
-}
-
-fn unix_like_cmd() {
-    use std::{thread, time};
-
-    let p = std::process::Command::new("sleep")
-        .arg("3")
-        .spawn()
-        .unwrap();
-    // To ensure that the system data are filled correctly...
-    thread::sleep(time::Duration::from_millis(250));
-    let mut s = sysinfo::System::new();
-    assert!(s.processes().is_empty());
-    s.refresh_processes();
-    assert!(!s.processes().is_empty());
-    let process = s.process(p.id() as sysinfo::Pid).unwrap();
-    if process.cmd() != ["sleep", "3"] {
-        panic!("cmd not equivalent to`[sleep, 3]`: {:?}", process);
-    }
-}
-
-#[test]
-#[cfg(not(windows))]
 fn test_environ() {
-    if !sysinfo::System::IS_SUPPORTED {
+    if !sysinfo::System::IS_SUPPORTED || cfg!(feature = "apple-sandbox") {
         return;
     }
+    let mut p = if cfg!(target_os = "windows") {
+        std::process::Command::new("waitfor")
+            .arg("/t")
+            .arg("3")
+            .arg("EnvironSignal")
+            .stdout(std::process::Stdio::null())
+            .env("FOO", "BAR")
+            .env("OTHER", "VALUE")
+            .spawn()
+            .unwrap()
+    } else {
+        std::process::Command::new("sleep")
+            .arg("3")
+            .stdout(std::process::Stdio::null())
+            .env("FOO", "BAR")
+            .env("OTHER", "VALUE")
+            .spawn()
+            .unwrap()
+    };
 
-    let mut p = std::process::Command::new("sleep")
-        .arg("3")
-        .stdout(std::process::Stdio::null())
-        .env("FOO", "BAR")
-        .env("OTHER", "VALUE")
-        .spawn()
-        .unwrap();
     let pid = p.id() as sysinfo::Pid;
     std::thread::sleep(std::time::Duration::from_millis(250));
     let mut s = sysinfo::System::new();
     s.refresh_processes();
-    p.kill().unwrap();
+    p.kill().ok();
```

Comment:
Why this change btw?

Reply:
I changed this because if the process fails to start, then this `unwrap()` would panic with a confusing error. I would rather get the subsequent assertions fail instead. In that case it's a bit more clear what the problem is. I guess I could also change this to a `.expect()`

Reply:
Please update it to a `expect` then. :)

Also please clean your commit history haha.

Reply:
Done! I squashed the commits and added expect everywhere.

Reply:
Thanks! Just waiting for CI then.

Reply:
Oh sorry, I just realized that I forgot to push the change to expect. So I just pushed again.